build.gradle

```diff
@@ -18,46 +24,85 @@ repositories {
     jcenter()
 }
 
+publishing {
+    publications {
+        customLibrary(MavenPublication) {
+            from components.java
+        }
+    }
+
+    repositories {
+        maven {
+            name = 'mt.edu.um.cf2.jgribx'
+            url = layout.buildDirectory.dir("repo")
+        }
+    }
```

Comment:
What's the intention behind publishing to a "repo" directory within the `/build` directory? If anything, we might want to publish to maven local for consumption by other local projects (e.g. MRHAB locally).

Eventually, we may also want to setup actual publishing to somewhere like Maven Central if possible (though we can also manually deploy to CTI Artifactory in the meantime).